mkt/extensions/models.py

```diff
@@ -153,7 +156,19 @@ def is_public(self):
 
     @property
     def manifest_url(self):
-        raise NotImplementedError
+        return absolutify(reverse('extension.manifest',
+                                  kwargs={'uuid': self.uuid}))
+
+    @property
+    def mini_manifest(self):
+        """Mini-manifest used for install/update in dict form.
+
+        Actually a maxi-manifest for the moment. Must contain `package_path`,
+        that points to the extension download absolute URL."""
```

Comment:
So, just to leave a trace somewhere: this will actually not be enough for installing to work. FxOS wants mini-manifests to be like apps, in a different format than the one we use for extensions. This means we'll have to create it from scratch using the data on the instance/actual manifest. I'll do that tomorrow.